ETC.tsv

```diff
@@ -16208,8 +16208,8 @@ ETC_20151022_016209	Crystal Mine Mission
 ETC_20151022_016210	Catacombs Mission
 ETC_20151022_016211	Fencer
 ETC_20151022_016212	The Fencer class makes use of the Rapier weapon to rapidly hit successive strikes. 
-ETC_20151022_016213	Nestos' Camp
-ETC_20151022_016214	Anastos' Camping Ground
+ETC_20151022_016213	Nestospa' Camp
```

Comment:
Nestopa Camp or Nestopa's Camp? Same for next line. 
' is only used when the name ends with a 's'.